index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    <h1>Hello Microverse!</h1>
+</body>
```

Comment:
- [ ] I can see that you have used a class `header` in your `.css` file to style your component, but the class is missing here (in your `html`). Let's make sure to add a class so that we have a great design 😄 Currently, your styles have no effect